.github/workflows/cli-build-instructions.yml

```
@@ -146,3 +146,64 @@ jobs:
+      - name: Run command - Generate Enums Test
+        run: |
+          cd unicodetools/mine/src
+          # run GenerateEnums
+          mvn -s .github/workflows/mvn-settings.xml exec:java -Dexec.mainClass="org.unicode.props.GenerateEnums"  -Dexec.args=""  -pl unicodetools  -DCLDR_DIR=$(cd ../../../cldr/mine/src ; pwd)  -DUNICODETOOLS_GEN_DIR=$(cd ../Generated ; pwd)  -DUNICODETOOLS_REPO_DIR=$(pwd)  -DUVERSION=15.0.0
+          # TODO(#100) Get rid of need for fake version 13.1 of Unicode enum val
+          # Fail if we haven't committed changes from Generate Enums, but make exception for fake Unicode version hack (#100)
+          echo "Check if GenerateEnums output has been committed to repo"
+          echo "git diff --compact-summary"
+          if git diff --compact-summary --exit-code; then
+            echo "Success: Any output from GenerateEnums has already been committed."
+            echo "exit 0"
+          else
+            echo "git diff --name-status | awk '{print $2;}'"
+            names=`git diff --name-status | awk '{print $2;}'`
+            if [[ "$names" == "unicodetools/src/main/java/org/unicode/props/UcdPropertyValues.java" ]]; then
+              gitdiffstat=`git diff --stat | tail -n 1`
+              if [[ "$gitdiffstat" = " 1 file changed, 1 insertion(+), 1 deletion(-)" ]]; then
+                git diff | grep 'V13_1("13.1")'
+                if [ $? -eq 0 ]; then
+                  echo "Inferred: Only non-committed change after GenerateEnums is temporary V13_1 hack (issue #100)"
+                  echo "Check passed."
+                  exit 0
+                fi
+              fi
+            fi
+            echo "Failure: GenerateEnums has changes that need to be committed."
+            exit 1
+          fi
+          # TODO(#100): Note: when running locally on a clean checkout, this command
+          # leaves a change in UcdPropertyValues.java by not preserving the comment 
+          # about the fake Unicode version 13.1.
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      # https://github.com/unicode-org/unicodetools/blob/main/docs/security.md#generating
+      - name: Run command - Generate Confusables
+        run: |
+          cd unicodetools/mine/src
+          # run GenerateConfusables
+          mvn -s .github/workflows/mvn-settings.xml exec:java -Dexec.mainClass="org.unicode.text.UCD.GenerateConfusables"  -Dexec.args="-c -b"  -pl unicodetools  -DCLDR_DIR=$(cd ../../../cldr/mine/src ; pwd)  -DUNICODETOOLS_GEN_DIR=$(cd ../Generated ; pwd)  -DUNICODETOOLS_REPO_DIR=$(pwd)  -DUVERSION=15.0.0
+          # run Build & Test command again to rerun TestSecurity test to verify output is okay
+          MAVEN_OPTS="-ea" mvn -s .github/workflows/mvn-settings.xml package -DCLDR_DIR=$(cd ../../../cldr/mine/src ; pwd)  -DUNICODETOOLS_GEN_DIR=$(cd ../Generated ; pwd)  -DUNICODETOOLS_REPO_DIR=$(pwd)  -DUVERSION=15.0.0
```

optional: Would it be easy to run only TestSecurity, not all tests? Something like this?
```suggestion
          MAVEN_OPTS="-ea" mvn -s .github/workflows/mvn-settings.xml test -Dtest=org.unicode.test.TestSecurity -DCLDR_DIR=$(cd ../../../cldr/mine/src ; pwd)  -DUNICODETOOLS_GEN_DIR=$(cd ../Generated ; pwd)  -DUNICODETOOLS_REPO_DIR=$(pwd)  -DUVERSION=15.0.0
```

---

Done.  The test lives in the Maven submodule called `unicodetools`.  Because we have a multi-module structure for our Maven project, the best way to invoke that is to run the command from the root, but then specify the submodule that we are really operating in, via `-pl unicodetools`.